ibm_cloud_sdk_core/base_service.py

```diff
@@ -245,6 +249,9 @@ def send(self, request: requests.Request, **kwargs) -> DetailedResponse:
             logging.exception('Error in service call')
             raise
 
+    def enable_gzip_compression(self, should_enable_compression: bool = False) -> None:
```

Comment:
```suggestion
    def set_enable_gzip_compression(self, should_enable_compression: bool = False) -> None:
```
And I think it would be good if we also had a getter for this flag as well.